src/SourceLocator/Exception/EmptyPhpSourceCode.php

```diff
@@ -1,11 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Roave\BetterReflection\SourceLocator\Exception;
-
-use InvalidArgumentException;
-
-class EmptyPhpSourceCode extends InvalidArgumentException
```

Comment:
I think we deprecate and remove this in the next major

Reply:
So one PR for 5.7 with `@deprecated` and this one should target 6.0.0?

Reply:
as you prefer on that: next major release branch is not yet open

Reply:
I would be happy to just open 6.0 branch so I can target PR for #932, #981 and #1073

Reply:
Works for me: beware that automatic releases will stop creating 5.x branches, once the new major branch is here

Reply:
Is there a way how I can open the branch? Or will you please do it for me? :)

Reply:
Oh, it already exists: https://github.com/Roave/BetterReflection/tree/6.0.x

Never mind 😁